app/views/admin/settings/project_custom_fields/new.html.erb

```diff
@@ -26,7 +26,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See COPYRIGHT and LICENSE files for more details.
 
 ++#%>
-<% content_controller 'admin--custom-fields', dynamic: true %>
+<% content_controller 'admin--custom-fields', dynamic: true, 'admin--custom-fields-format-config-value': custom_fields_format_config %>
```

Comment:
I think this could be moved into the _form partial so that it only gets called once. Or is there any field I am not aware  of outside of the form that is also controlled by the stimulus controller

Reply:
Indeed, also I didn't see why is it needed to set stimulus controller for the content wrapper and not just for the form

Reply:
821c21e978a634e34b739e2642bc4d78e24f3700 and also e19f46960f1d2d92ddf2b961a2404705d9091862